dist/bin/dotd

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
+  ## resolve links - $0 may be a link to PROG_HOME
+  PRG="$0"
+
+  # need this for relative symlinks
+  while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+      PRG="$link"
+    else
+      PRG="`dirname "$PRG"`/$link"
+    fi
+  done
+
+  saveddir=`pwd`
+
+  PROG_HOME=`dirname "$PRG"`/..
+
+  # make it fully qualified
+  PROG_HOME=`cd "$PROG_HOME" && pwd`
+
+  cd "$saveddir"
+fi
+
+source "$PROG_HOME/bin/common"
+
+default_java_opts="-Xmx768m -Xms768m"
+bootcp=true
+
+PROG_NAME=dotty.tools.dottydoc.Main
```

Comment:
@felixmulder could you give me a tip on how the dottydoc script works? The main difference from `dotc` is just the main method?

I run `dotd examples/hello.scala` locally, but it seems we need more libraries: `java.lang.NoClassDefFoundError: com/vladsch/flexmark/util/options/MutableDataSet`.